fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1666,6 +1666,110 @@ module.exports = function(config, DB) {
         )
 
         test(
+          'securityEvents',
+          function (t) {
+
+            function securityEvents(suite) {
```

Comment:
This took me a while to get my head round for some reason but I really like it, nice one. 👍